plugins/jira-dashboard/src/components/JiraTable/columns.tsx

```diff
@@ -130,9 +136,13 @@ export const columnAssignee: TableColumn<Issue> = {
               height: 35,
             }}
           />
-          <Typography variant="body2">
-            {issue.fields.assignee.displayName}
-          </Typography>
+          <EntityPeekAheadPopover
```

Comment:
Can we show the EntityPeekAheadPopover even if only issue.fields?.assignee?.name or key is available (and not displayname)?